droidlet/memory/sql_memory.py

```diff
@@ -609,6 +620,26 @@ def get_most_recent_incoming_chat(self, after=-1) -> Optional["ChatNode"]:
         else:
             return None
 
+    ###################
+    ## Logical form ###
+    ###################
+
+    def add_logical_form(self, logical_form: dict):
```

Comment:
similarly... I think we should be moving just using the ProgramNode.create

Reply:
same as above ^

Reply:
also fine